mdtraj/geometry/distance.py

```diff
@@ -86,6 +85,61 @@ def compute_distances(traj, atom_pairs, periodic=True, opt=True):
         return _distance(xyz, pairs)
 
 
+def compute_distances_t(traj, atom_pairs, time_pairs, periodic=True, opt=True):
+    """Compute the distances between pairs of atoms from times 0 to t.
+
+    Parameters
+    ----------
+    traj : Trajectory
+        An mtraj trajectory.
+    atom_pairs : np.ndarray, shape=(num_pairs, 2), dtype=int
+        Each row gives the indices of two atoms involved in the interaction.
+    time_pairs : nd.array
+        Each row gives the indices of two frames.
+    periodic : bool, default=True
+        If `periodic` is True and the trajectory contains unitcell
+        information, we will compute distances under the minimum image
+        convention.
+    opt : bool, default=True
+        Use an optimized native library to calculate distances. Our optimized
+        SSE minimum image convention calculation implementation is over 1000x
+        faster than the naive numpy implementation.
+
+    Returns
+    -------
+    distances : np.ndarray, shape=(n_frames, num_pairs), dtype=float
```

Comment:
Reading this docstring, without first having read all of the code, I'm not sure what the shape of the return value is. If there are multiple time pairs (which I'm inferring from the fact that the argument is pluralized // time "pairs" rather than time "pair"), when why isn't the return value 3-dimensional (num_frames x num_atom_pairs x num_time_pairs)?

Reply:
Oh no, it's (num_time_pairs x num_atom_pairs). My misunderstanding. Anyways the docstrings should be clarified.